docs/guide/api-hmr.md

```diff
@@ -10,21 +10,31 @@ Vite exposes its manual HMR API via the special `import.meta.hot` object:
 
 ```ts
 interface ImportMeta {
-  readonly hot?: {
-    readonly data: any
+  readonly hot?: ViteHotContext
+}
+
+interface ViteHotContext {
+  readonly data: any
+
+  accept(): void
+  accept(cb: (mod: any) => void): void
+  accept(dep: string, cb: (mod: any) => void): void
+  accept(deps: readonly string[], cb: (mods: any[]) => void): void
 
-    accept(): void
-    accept(cb: (mod: any) => void): void
-    accept(dep: string, cb: (mod: any) => void): void
-    accept(deps: string[], cb: (mods: any[]) => void): void
+  /**
+   * @deprecated
+   */
+  acceptDeps(): never
```

Comment:
I think we should not put deprecated function as the current documentation doesn't have any deprecated api and Vite 3 is also in preparation.

Reply:
Thanks for your review